### ISSUE TITLE
Table column resize produces errors for the table with all the header rows

### ISSUE THREAD

**Issue by Dumluregn** [ckeditor/ckeditor5#11878]
## 📝 Provide detailed reproduction steps (if any)

1. Go to https://ckeditor.com/docs/ckeditor5/latest/features/table.html#table-column-resize
2. Select all the rows
3. Change them to header rows
4. Try to resize any column

### ✔️ Expected result

Resize works normally.

### ❌ Actual result

There are errors in the console:
```
snippet.js:4 Uncaught TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
    at wk (snippet.js:4:729485)
    at bk (snippet.js:4:729722)
    at Rk._getResizingData (snippet.js:4:738645)
    at Rk._onMouseDownHandler (snippet.js:4:736448)
    at Object.fire (snippet.js:4:2958)
    at Us (snippet.js:4:70810)
    at Gs.fire (snippet.js:4:70316)
    at bu.fire (snippet.js:4:122423)
    at bu.onDomEvent (snippet.js:4:394164)
    at listenTo.useCapture (snippet.js:4:122332)
rethrowUnexpectedError @ snippet.js:4
fire @ snippet.js:4
Us @ snippet.js:4
fire @ snippet.js:4
fire @ snippet.js:4
onDomEvent @ snippet.js:4
listenTo.useCapture @ snippet.js:4
fire @ snippet.js:4
t @ snippet.js:4
snippet.js:4 Uncaught TypeError: Cannot destructure property 'columnPosition' of 'this._resizingData' as it is null.
    at Rk._onMouseMoveHandler (snippet.js:4:737512)
    at m (snippet.js:4:123458)
    at g (snippet.js:4:123514)
    at ur.b (snippet.js:4:123836)
    at ur.fire (snippet.js:4:2958)
    at HTMLDocument.t (snippet.js:4:115818)
```

## 📃 Other details

* Browser: all
* OS: macOS
* First affected CKEditor version: `34.1.0` (the first one with column resize feature)
* Installed CKEditor plugins: `table`

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by Dumluregn** [comment]
Some valuable input is here - https://github.com/ckeditor/ckeditor5/pull/11914#issuecomment-1169626373.

**Comment by mlewand** [comment]
Fixed in #12057
